web/privacy.html

```diff
@@ -1,19 +1,22 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="it">
 
 <head>
   <meta charset="utf-8">
-  <meta http-equiv="x-ua-compatible" content="ie=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
```

Comment:
Why this change?

Reply:
The last attribute ("shrink-to-fit=no") serves the purpose to improve viewport's rendering from mobile. In a nuthsell: viewport meta tags having "width=device-width" cause the page to scale down to fit content that overflows the viewport bounds. You can override this behavior by adding "shrink-to-fit=no", preventing the page from scaling to fit the viewport.
Such added value is compliant with Bootstrap Italia's library requirements(see [here](https://italia.github.io/bootstrap-italia/docs/come-iniziare/introduzione/#utilizzo-come-dipendenza) under Meta tag responsive)

Reply:
@balanza yes I removed intentionally. The tag itself serves the purpose to improve rendering for older versions of IE. I removed it because I did not find any reference to this attribute in Bootstrap Italia's library. It is used in io.italia.it homepage because (I suppose) it is possible that the public at large may use such older versions of IE. I am reinserting it in the new PR though